src/lib/parse_org.js

```diff
@@ -809,3 +804,38 @@ export const parseOrg = fileContents => {
     linesBeforeHeadings,
   });
 };
+
+const extractActiveTimestampsForPlanningItemsFromParse = (type, parsedData) => {
+  // planningItems only accept a single timestamp -> ignore second timestamp
+  return parsedData
+    .filter(x => x.get('type') === 'timestamp' && x.getIn(['firstTimestamp', 'isActive']))
+    .map(x => createTimestamp({ type: type, timestamp: x.get('firstTimestamp') }));
+};
+
+// Merge planningItems from parsed title, description, and planning keywords.
+const mergePlanningItems = (...planningItems) => {
+  return planningItems[0].concat(...planningItems.slice(1));
+};
+
+export const updatePlanningItems = (planningItems, type, parsed) =>
```

Comment:
Is there a style guide or best practice to always use hashs for more than 1 or 2 arguments?

Like `({planningItems, type, parsed}) => ...`

Or what is your opinion on this, @munen ?

Reply:
Yes, using destructuring for function parameters is very common. It has a couple benefits:

- JS doesn't have 'named parameters', but with destructuring you get the same names everywhere
- Changing position doesn't require refactoring in potentially many places (which is always hard in a dynamically typed language)
- It's easy to add new properties to data and it's explicit to remove them

It's also common in other languages like Clojure.

Personally, I prefer to use destructuring when it's more than two parameters, for these reasons. And some others do, too. For example there's an eslint rule for that: https://eslint.org/docs/rules/prefer-destructuring

Having said that, I'm not nitting on a function with 3 parameters. If that's what you prefer, it's fine with me.